pal/system-info/minigame/system-info.ts

```diff
@@ -125,13 +127,13 @@ class SystemInfo extends EventTarget {
 
         this.isXR = false;
 
-        const isPCWechat = WECHAT && this.os === OS.WINDOWS && !minigame.isDevTool;
+        const isPCWechat = (WECHAT || WECHAT_MINI_PROGRAM) && this.os === OS.WINDOWS && !minigame.isDevTool;
```

Comment:
we don't support PC wechat mini program, so please remove this condition

Reply:
ok.